src/containerapp/azext_containerapp/_params.py

```diff
@@ -347,3 +352,34 @@ def load_arguments(self, _):
 
     with self.argument_context('containerapp env', arg_group='Peer Traffic Configuration') as c:
         c.argument('p2p_encryption_enabled', arg_type=get_three_state_flag(), options_list=['--enable-peer-to-peer-encryption'], is_preview=True, help='Boolean indicating whether the peer-to-peer traffic encryption is enabled for the environment.')
+    
+    with self.argument_context('containerapp sessionpool') as c:
+        c.argument('name', options_list=['--name', '-n'], help="The Session Pool name.")
+        c.argument('resource_group_name', arg_type=resource_group_name_type, id_part=None)
+
+    with self.argument_context('containerapp sessionpool', arg_group='Configuration') as c:
+        c.argument('container_type', arg_type=get_enum_type(["CustomContainer", "PythonLTS"]), help="The pool type of the Session Pool, default=PythonLTS")
+        c.argument('cooldown_period', help="Period (in seconds), after which the session will be deleted, default=300")
+        c.argument('secrets', nargs='*', options_list=['--secrets', '-s'], help="A list of secret(s) for the session pool. Space-separated values in 'key=value' format.")
```

Comment:
how to clear all secrets?

Reply:
By setting to empty string, it not documented in the description, but it work in this way.

Reply:
Update the description